presto-main/src/main/resources/webapp/src/components/LivePlan.jsx

```diff
@@ -125,10 +125,19 @@ class PlanNode extends React.Component<PlanNodeProps, PlanNodeState> {
     }
 
     render() {
+        const details = this.props.details;
+        var distribution = "";
+
+        if (details.includes("Distribution")) {
```

Comment:
Perhaps you can try matching against a regex and get the distribution type from a match group.